templates/deployment-worker.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: worker
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: airflow
+        tier: worker
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: worker
+        image: eanylin/airflow-helm:1.8.1-1.6.2
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+          - name: dags
+            mountPath: /usr/local/airflow/dags
+          - name: docker
+            mountPath: /var/run
+            readOnly: false
+          - name: pod-var-lib-docker
+            mountPath: /var/lib/docker
+            readOnly: false
+        env:
```

Comment:
This is really a global comment for all ENVs, do we actually need the `env` variables such as this, if helm is driving airflow.cfg?